File: 프로그래머스/0/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.py
def solution(arr):
    answer = []
    for i in arr: 
         answer += [i] * i
    return answer
""" 양의 정수 배열 arr가 주어지고 찰예대로 원소를 보면서 들어온 수 만큼 넣어주라 """

File: 프로그래머스/0/181861. 배열의 원소만큼 추가하기/배열의 원소만큼 추가하기.js
function solution(arr) {
    var answer = [];
      for (let i = 0; i < arr.length; i++) {
        for (let j = 0; j < arr[i]; j++) {
            answer.push(arr[i]);
        }
    }
    return answer;
}

//들어온 값을  그 숫자만큼 repeat하기 
